### PR TITLE
[COREAPI-396] remove restid-related helper, jwt generator, refactor logger

### LINKED ISSUE #50: ElasticSearch Client nil Panic
func `get` on `v1/middleware/auth.go` will panic if `*elastic.Client` (on this file, the variable name is `es`).
```go
searchResult, err := es.Search().
		Index("users").
		Type("_doc").
		Query(query).
		Do(context.Background())
```
need to check if provided `*elastic.Client` is not nil.
image

### PR DESCRIPTION
**Pull request by heggies**
## Description

> Please add your task in ClickUp here.

Task: [COREAPI-396](https://app.clickup.com/t/3857515/COREAPI-396)

## What's new?

> Please add if you added new improvement
- remove `restid`-related helper
- remove jwt generator
- refactor logger, stack trace
- fix #50

### CODE REVIEW

File: v1/logger/logger.go [hunk 1]
```diff
@@ -45,52 +52,39 @@ func realIP(req *http.Request) string {
...
-		return
-	}
-	logger.Hooks.Add(hook)
-
-	start := time.Now()
-	latency := time.Since(start)
-	fields["ResponseTime"] = latency
+// log params
+// 	@fields: logrus.Fields
+// 	@errMsg: string
```
Comment (ahtuz) [inline v1/logger/logger.go:57]: private func perlu godoc?
@heggies @christyyy-a

**Review by ahtuz — CHANGES_REQUESTED** [review]

File: v1/logger/logger.go (hunk 1 above)
Comment (heggies) [reply v1/logger/logger.go:57]: bagaimana @christyyy-a 
saya ngikutin christy

Comment (christyyy-a) [reply v1/logger/logger.go:57]: image
semoga menjawab

**Review by christyyy-a — APPROVED** [review]

**Review by ahtuz — APPROVED** [review]

File: v1/logger/logger.go (hunk 1 above)
Comment (ahtuz) [reply v1/logger/logger.go:57]: jadi jawabnya apa ini WKWKK
pakein aja?

Comment (agusID) [reply v1/logger/logger.go:57]: @heggies , ini dipakai aja, GTG

**Review by agusID — APPROVED** [review]

**Review by yafetzz — APPROVED** [review]

**Pull request merged**